### DISCUSSION TITLE
Updating PR comment fails when migrating Renovate user

### DISCUSSION THREAD

**Discussion by fgreinacher** [renovatebot/renovate#27411] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

We are migrating from one Renovate bot user to another one. We have configured https://docs.renovatebot.com/configuration-options/#ignoreprauthor as suggested.

Renovate updates old MRs as expected, but it fails in case where the MR has an old comment that needs to be updated (e.g. artifact update errors). Editing comments is only possible by the author or project maintainers.

I assume a similar issues exists on other platforms, but have not verified.

### Relevant debug logs

<details><summary>Logs</summary>

```
"stack": "HTTPError: Response code 403 (Forbidden)\n    at Request.<anonymous> (/opt/containerbase/tools/renovate/37.175.0/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)",
"options": {
  "headers": {
    "user-agent": "RenovateBot/37.175.0 (https://github.com/renovatebot/renovate)",
    "accept": "application/json",
    "authorization": "***********",
    "content-type": "application/json",
    "content-length": "961",
    "accept-encoding": "gzip, deflate, br"
  },
  "url": "https://our-gitlab.com/api/v4/projects/a%2Fproject/merge_requests/131/notes/26605341",
  "hostType": "gitlab",
  "username": "",
  "password": "",
  "method": "PUT",
  "http2": false
},
```

</details>

### Have you created a minimal reproduction repository?

No, I wanted to start discussing this first and can set up a reproduction if needed, just let me know.

**Comment by rarkins** [comment]
Is this for an artifact error message?

**Reply by fgreinacher** [reply]
Yes

**Reply by viceice** [reply]
it seems you need to manually fix that, as we don't have rights to do.

is it a hard failure or does renovate still proceed?
